front/src/views/ArtworkView.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="Artwork">
+    <ArtWork :id="this.id_artwork" />
+  </div>
+</template>
+
+<script>
+import ArtWork from "@/components/Artwork.vue";
+
+export default {
+  components: {
+    ArtWork,
+  },
+  created() {
+    this.id_artwork = this.$route.params.id_artwork;
```

Comment:
Going to http://localhost:3000/artwork/test causes errors

Check the route params before instancing the Artwork, if id doesn't exists, return to homepage.